File: C/BaekJoon/Bronze/B2_32860.c
#include <stdio.h>

int main() {
  int year, number;
  scanf("%d %d", &year, &number);

  printf("SN ");
  printf("%d", year);

  if (number < 27) {
    printf("%c", number + 64);
  } else {
    int q = (number - 1) / 26;
    int r = (number - 1) % 26;

    printf("%c", 'a' + q - 1);
    printf("%c", 'a' + r);
  }

  return 0;
}

File: Java/BaekJoon/Bronze/B2_32860.java
package BaekJoon.Bronze;

import java.util.Scanner;

public class B2_32860 {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);

        int year = sc.nextInt();
        int number = sc.nextInt();

        System.out.print("SN ");
        System.out.print(year);

        if (number < 27) {
            System.out.print((char)(number + 64));
        } else {
            int q = (number - 1) / 26;
            int r = (number - 1) % 26;

            System.out.print((char)('a' + q - 1));
            System.out.print((char)('a' + r));
        }

        sc.close();
    }
}
